docs/source/requirements_docs.txt

```diff
@@ -1,6 +1,8 @@
+dill
+ipython
+ipykernel
 nbsphinx
+python-dotenv[cli]
 sphinx_copybutton
 sphinx_rtd_theme
-python-dotenv[cli]
-ipykernel
```

Comment:
This is required for notebooks to actually run while building on readthedocs

Reply:
I alphabetized for maintainability. The original commit just adds `dill`, `ipython`, and `ipykernel`.

Reply:
hi Eric, actually `dill` can be dropped as an explicit requirement, since RP is in requirements and `dill` is in its dependencies (since 1.14 - https://github.com/radical-cybertools/radical.pilot/blob/devel/setup.py#L270)

Reply:
Done.

I just tried `make html` in a fresh environment (after setting the DBURL) and it got as far as `reading sources... [ 21%] 3/3.2-obtaining-task-details `. It's been there for a couple of minutes. I haven't bothered to check what it's doing, but I'm assuming it's got at least far enough to confirm the Python package set.

Does it have to do actual RP work to build docs every time? Does nbsphinx have a way to mock imports, the way autodoc can?